release/Magarena/scripts/Aetherstorm_Roc.groovy

```diff
@@ -1,37 +1,47 @@
 def action = {
     final MagicGame game, final MagicEvent event ->
-    game.doAction(new ChangeCountersAction(event.getPermanent(), MagicCounterType.PlusOne, 1));
     if (event.isYes()) {
-        event.processTargetPermanent(game, {
-            game.doAction(new TapAction(it));
-        });
+        game.doAction(new ChangeCountersAction(event.getPlayer(), MagicCounterType.Energy, -2));
+        game.doAction(new ChangeCountersAction(event.getPermanent(), MagicCounterType.PlusOne, 1));
+        game.doAction(new TapAction(event.getRefPermanent()));
```

Comment:
probably not a good idea to operate on MagicPermanent.NONE, please check isValid before performing tap action